wadl/solver/solver.py

```diff
@@ -85,7 +86,12 @@ def setup(self, graph):
     def solve(self, routeSet):
         startTime = time.time()
         # solve each tile
+        # save subPaths, metaGraph after initially found so that they can be reused
         subPaths = self.solveTiles()
+        self.subPathsCopy = copy.deepcopy(subPaths)
```

Comment:
you don't need to deepcopy this object you can just change the above line to `self.pubpaths = ...`